### PR TITLE
[Console][Process] do not let context classes extend the message classes

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | 
| License       | MIT

### CODE REVIEW

**Comment by xabbuh** [comment]
minor tweak after talking with @kbond about #49813

**Review by kbond — APPROVED** [review]

**Comment by xabbuh** [comment]
also, @kbond is right in that we can make a similar change in `RunCommandContext` (introduced with #49814)

File: src/Symfony/Component/Console/Messenger/RunCommandContext.php [hunk 1]
```diff
@@ -14,10 +14,9 @@
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  */
-final class RunCommandContext extends RunCommandMessage
+final class RunCommandContext
 {
-    public function __construct(RunCommandMessage $message, public readonly int $exitCode, public readonly string $output)
+    public function __construct(public readonly RunCommandMessage $message, public readonly int $exitCode, public readonly string $output)
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Console/Messenger/RunCommandContext.php]: let's split these on many lines all with final comma

File: src/Symfony/Component/Process/Messenger/RunProcessContext.php [hunk 2]
```diff
@@ -16,16 +16,14 @@
...
  */
-final class RunProcessContext extends RunProcessMessage
+final class RunProcessContext
 {
     public readonly ?int $exitCode;
     public readonly ?string $output;
     public readonly ?string $errorOutput;
 
-    public function __construct(RunProcessMessage $message, Process $process)
+    public function __construct(public readonly RunProcessMessage $message, Process $process)
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Process/Messenger/RunProcessContext.php]: let's declare the property separately to make reading easier

**Comment by nicolas-grekas** [comment]
Thank you @xabbuh.

**Pull request merged**